ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -377,33 +377,6 @@ public struct Avatar: View {
 
     private let animationDuration: Double = 0.1
 
-    private struct PresenceCutout: Shape {
```

Comment:
Seems like this PR will conflict with #921 where you are moving this struct around. Should we just do the naming cleanup in that branch and migrate it to `main` later, or is there benefit to getting this change published sooner?

Reply:
I think there is benefit in having less code in main now and this change can just be integrated to the #921 branch once it's merged.

Reply:
I think moving it to a new file at least makes it easier to deal with the merge fallout.